README.md

```diff
@@ -1,2 +1,155 @@
-# Hello-world
-First Project Module 1 Week 1 Microverse Course
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <img src="LOGO.jpg" alt="logo" width="160"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Project</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+
+# 📖 Hello World <a name="about-project"></a>
```

Comment:
- [ ] While you did a wonderful job :+1:, kindly remember to include the MIT `LICENSE` in your project and Link it to your `README` file as the [template](https://github.com/microverseinc/readme-template) shows. This is important because it clearly outlines the terms and conditions for others to use and contribute to your code, thereby protecting your intellectual property rights. Failure to include the license could result in legal issues or disputes down the line, and may also deter potential users and collaborators from using or contributing to your project :+1:. You can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

Reply:
Hello Ing. David! Thank you very much for taking the time to review my project, I added the License file and I hope the project is successfully finished.  Please, let me know if there is another issue to be fixed. 

Greetings have a nice day!